clusters/hosted-mgmt/hive/pools/rh-ecosystem-edge/ocp-4-13-0_cluster-pool.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: 4-13-cluster-pool
+  namespace: rh-ecosystem-edge-cluster-pool
+  labels: # architecture, cloud, owner, product, version are used to filter out a pool when a job claims a cluster
+    architecture: amd64
+    cloud: aws
+    product: ocp
+    owner: rh-ecosystem-edge
+    version: "4.13"
+    version_lower: 4.13.0-0 # optional: lower bound for automatically updated imageset; required if version_upper is set
+    version_upper: 4.14.0-0 # optional: upper bound for automatically updated imageset; required if version_lower is set
+spec:
+  baseDomain: rh-ecosystem-edge.openshift.org # the base domain to install the cluster
+  imageSetRef:
+    name: ocp-release-4.13.43-x86-64-for-4.13.0-0-to-4.14.0-0 # the name of the imageSet which determines the image to install the cluster; will be automatically updated if `version_*` labels are set
+  installConfigSecretTemplateRef:
+    name: rh-ecosystem-edge-install-config # the name of the secret with an installation config for the installer
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: aws-credentials # the name of the secret with the credentials of the cloud account
+      region: us-east-1
+  pullSecretRef:
+    name: pull-secret
+  size: 2 # the number of clusters that Hive should keep provisioned and waiting for use.
+  maxSize: 3 # the maximum number of clusters that can exist at the same time.
```

Comment:
Then this is either 3 or 6.

Reply:
This is 3 with the option to install 3 more on demand if when the pool size is lower than 3.